CodeforAustralia/council-emissions-calculator
# fix typo in svg filename

## Overview of the Pull Request:
This PR fixes a bug introduced in #156 that could have caused the results page not to load.
Fix was simply to address a typo in filename of an svg file.

## link to Trello card or Github issue

## How Has This Been Tested?
- verify that the results page loads correctly in preview deployment
- check the [deployment log](https://vercel.com/codeforaustralia/council-emissions-calculator-spike/78Wh7aZmRxevG5eYEkopvXQ6Uw4g) that there are no error messages

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!
